shared_libraries/orca_shared/database/adapters/api/aws.py

```diff
@@ -1,15 +1,18 @@
 import json
-import logging
 import os
 
 import boto3
+from aws_lambda_powertools import Logger
 
 from orca_shared.database.entities.postgres_connection_info import (
     PostgresConnectionInfo,
 )
 
+# Set AWS powertools logger
+Logger = Logger()
 
-def get_configuration(db_connect_info_secret_arn: str, logger: logging.Logger) \
+
+def get_configuration(db_connect_info_secret_arn: str, logger: Logger) \
```

Comment:
Not sure why we are passing `logger: Logger` here and not just using it because it is defined up above. I feel this should be removed as an argument possible. I would put in a ticket to revisit this. Not part of this PR though. This would look at the shared library overall with logger including use cases, etc.

Reply:
created https://bugs.earthdata.nasa.gov/browse/ORCA-568